oxen-rust/test-put-demo.sh

```
@@ -0,0 +1,477 @@
+fi
+
+echo "=== Oxen PUT Demo Script ==="
+echo "Using Oxen binaries from: $OXEN_TARGET_DIR"
+echo "Working directory: $WORKDIR"
+
+# Create and enter working directory
+echo
+echo "Setting up working directory: $WORKDIR"
+rm -rf "$WORKDIR"
+mkdir -p "$WORKDIR"
+cd "$WORKDIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)